src/main/java/com/twilio/sdk/resource/instance/taskrouter/WorkspaceStatistics.java

```diff
@@ -242,6 +246,31 @@ public Integer getTotalWorkers() {
 	}
 
 	/**
+	 * Get the number of tasks for each priority
+	 *
+	 * @return the number of tasks with priority p
+	 */
+	public int getTasksWithPriority(int priority) {
+		Map<String, Object> tasksByPriority = (Map<String, Object>) getRealtime().get(TASKS_BY_PRIORITY_PROPERTY);
```

Comment:
this feels a bit confusing - if I want to get a task count with a single priority I would use an `int` here, but if I do it via the `Map` below, I would use a `String`? Since the data returns the `priority` value as a `String` should we do fetching based on that as well?